.changeset/shy-ravens-prove.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sap-ux/project-access': minor
+---
+
+Introduce `readUi5Yaml` to read ui5.yaml files
```

Comment:
add might be a better choice than introduce here

Reply:
Maybe a native speaker needs to comment here. It is not wrong to use `introduce` but it sounds off (to me)

Reply:
Changed to `add`. Only part is `Introducing` for app-config-writer, but can be changed too, if necessary.